public/js/gravityexport-lite.js

```diff
@@ -65,6 +65,30 @@ var gfexcel_sortable;
     };
 
     $(document).ready(function () {
-        $("#start_date, #end_date").datepicker({dateFormat: 'yy-mm-dd', changeMonth: true, changeYear: true});
+		$( '#start_date, #end_date' ).datepicker( { dateFormat: 'yy-mm-dd', changeMonth: true, changeYear: true } );
+
+		$( '#file_extension' ).on( 'change', function ( e ) {
+			const shortcode = $( '#embed_code' ).val();
+
+			$( '#embed_code' ).val( shortcode.replace( /type=\"[^\"]*\"/, `type="${ e.target.value }"` ) );
+		} );
+
+		$( '#has_embed_secret' ).on( 'change', function () {
+			const $embedShortcodeEl = $( '#embed_code' );
+			const secret = $embedShortcodeEl.data( 'secret' );
```

Comment:
We can extract these from the function to avoid multiple calls for a `const` (micro optimisation)